lib/controller.rb

```diff
@@ -0,0 +1,15 @@
+class Controller
+  def initialize(board, display, game)
+    @board = board
+    @display = display
+    @game = game
+  end
+
+  def make_move(player)
+    @display.display_grid(@board.grid)
+    @display.display_make_move
+    move = @game.prompt_make_move
+    @board.make_move(player, move)
```

Comment:
Am I right that `make_move` in `Board` uses only player's mark, not a player? Passing an argument named `player` may suggest that there is something more (an object) than just a mark.

Reply:
You're right, it will use the mark only. Thank you, I'll change this and take this into consideration when creating the players